### PR TITLE
Display a chosen recipe

### PR DESCRIPTION
**Pull request by chloee122**
Added display of a chosen recipe feature.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,29 +2,57 @@ import { useState } from "react";
 import { searchRecipes, getRecipe } from "./api";
 import RecipeSearchForm from "./component/RecipeSearchForm";
 import RecipeList from "./component/RecipeList";
-import Recipe from "./component/Recipe";
+import DisplayRecipe from "./component/DisplayRecipe";
+import BookmarkList from "./component/BookmarkList";
 
 function App() {
   const [recipes, setRecipes] = useState([]);
+  const [currentPage, setCurrentPage] = useState(0);
```
Comment (duyle95) [inline src/App.js:10]: App.js file starts getting to big. I think you should refactor this file a bit. For example, moving this state into `RecipeList` component.

File: src/App.js [hunk 2]
```diff
@@ -2,29 +2,57 @@ import { useState } from "react";
...
+  const [showBookmarks, setShowBookmarks] = useState(false);
```
Comment (duyle95) [inline src/App.js:11]: Refactor idea: bookmarks logic can stay within a big "bookmark" component, which contains these components:
```
<button onMouseEnter={handleHover} onMouseLeave={handleLeave}>
          BOOKMARKS
</button>
{showBookmarks && <BookmarkList bookmarks={bookmarks} />
```

File: src/component/DisplayRecipe.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  },
+  onBookmark,
+}) {
+  const [servings, setServings] = useState(defaultServing);
+
+  useEffect(() => {
+    setServings(defaultServing);
+  }, [defaultServing]);
+
+  const content = (
```
Comment (duyle95) [inline src/component/DisplayRecipe.js:25]: No need to define a `content` having the all the React component here. This even makes it less readable and you are wrapping an extra `<div></div>`to the whole component 🤔 

You can also replace `div` with `React.Fragment`.

File: src/App.js (hunk 1 above)
Comment (chloee122) [reply src/App.js:10]: this has been fixed in the previous PR when you taught me about useEffect hihi

File: src/component/DisplayRecipe.js (hunk 3 above)
Comment (chloee122) [reply src/component/DisplayRecipe.js:25]: how about `return content` only?

**Pull request merged**